homeassistant/components/heatmiser/climate.py

```diff
@@ -41,36 +39,34 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the heatmiser thermostat."""
     from heatmiserV3 import heatmiser, connection
```

Comment:
Would be nice if you could move all imports to the file header.

Reply:
Done - I copied the light component example from the docs that has this pattern.  If the rule is that these should always be in the header - let me know and I'll submit a PR back to that example too.